gitnamehere/point-of-sale-group-project
# Cart display fixes

**PR Desc:**
- Fixed the bug where items in the cart would be shared amongst all users
- Converted from using the database `cart_item` to localStorage
- Added the feature to clear the cart after an order is placed

**Testing:**
- Run `npm i`
- Run `npm run setup-demo:dev`
- Run `npm run start:dev`
- Visit http://localhost:3000/store/menu/#
- Open up an incognito window and visit the same link
- Add items on one of the websites and refresh the other
- Verify that the one you refreshed does not contain the items you added
- Place an order and verify the cart is empty afterward
